src/comps/LogIn.jsx

```diff
@@ -16,9 +18,9 @@ export default function LogIn(props) {
             password: password,
           })
         const data = await res.data;
-        if (data.msg == 'success') {
+        if (data.msg === 'success') {
           nav('/products')
-          props.userConnect(data.user)
+          valContext.userConnect(data.user)
         } else {
```

Comment:
The userConnect name is not clear, sounds like a property of the valContext object but is actually a method. Consider changing this name into something better like `setUserConnected` for examlpe